fastkml/times.py

```diff
@@ -321,60 +280,14 @@ def __bool__(self) -> bool:
         """Return True if the begin or end date is valid."""
         return bool(self.begin) or bool(self.end)
 
-    def etree_element(
-        self,
-        precision: Optional[int] = None,
-        verbosity: Verbosity = Verbosity.normal,
-    ) -> Element:
-        """
-        Create an Element object representing the time interval.
 
-        Args:
-        ----
-            precision (Optional[int]): The precision of the time values.
-            verbosity (Verbosity): The verbosity level for the element.
+class When(_TimePrimitive):
```

Comment:
I think `When`, `Begin` and `End` should inherit from  `_BaseObject)`  not  `_TimePrimitive`

Reply:
This should be fine as `_BaseObject` is higher in the inheritance hierarchy while still having access to the class methods `etree_element` and `_get_kwargs` defined in the base class `_XMLObject`.
* These two class methods will be needed to implement the `When`, `Begin` and `End` classes.